### PR TITLE
Rewite login page to use component, nicer HTML styling.

### PR DESCRIPTION
**Pull request by ewels**
- Make the log in form on the homepage functional.
- Rewrite the dedicate login page to use the same flask component for the form used on the homepage
- Rewrite HTML to use nicer bootstrap formatting

Note that the validation stuff seems quite difficult to test, if I enter random stuff I get errors in the flask flash messages but nothing through the form. I guess I'll test it a bit more when I get to the other forms which have stricter validation criteria.

image

image

- [x] Tests passing
- [x] Black formatting
- [ - ] Migrations for any changes to the database schema
- [ - ] Rebase/merge the `dev` branch
- [ - ] Note in the CHANGELOG

### CODE REVIEW

File: dds_web/templates/home.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {% extends 'base.html' %}
 {% block body %}
 
+<div class="alert alert-info h4 text-center">
+    For information on how to use the Data Delivery System, as well as to see a testing protocol, see
```
Comment (ewels) [inline dds_web/templates/home.html:5]: This text is a meant to be temporary only and should be removed after testing..

File: dds_web/templates/user/login.html [hunk 2]
```diff
@@ -6,50 +6,16 @@
...
-            For information on how to use the Data Delivery System, as well as
-            see a test protocol, go <a
-                href="https://github.com/ScilifelabDataCentre/dds_cli/blob/master/docs/_build/latex/datadeliverysystem.pdf">here</a>.
+<p class="lead">Please use the form below to log in to the SciLifeLab Data Delivery System.</p>
+<div class="row">
+    <div class="col-lg-6">
+        {% include "components/login_form.html" %}
+    </div>
+    <div class="col-lg-6">
+        <p>Typically, you will arrive here because you have recieved an email from a unit that uses
```
Comment (ewels) [inline dds_web/templates/user/login.html:15]: This text is a bit crap, help writing something better would be appreciated 🙏🏻

Comment (inaod568) [reply dds_web/templates/user/login.html:15]:
```
In order to get a DDS account you need an invitation from one of the SciLifeLab units that use the DDS for data delivery. Once you have received the invitation you can register an account and log in to the system.
```
Or something like that?

Comment (ewels) [reply dds_web/templates/user/login.html]: Looks good 👍🏻 Applied..

**Review by inaod568 — APPROVED** [review]

**Pull request merged**